newsfragments/1918.internal.rst

```diff
@@ -0,0 +1,3 @@
+Added :cls:`~eth.db.accesslog.KeyAccessLoggerDB` and its atomic twin; faster ``make validate-docs``
```

Comment:
This needs to be `:class:` (I had to these up in the release notes yesterday, which is why know :sweat_smile: )